modules/installation-aws-permissions.adoc

```diff
@@ -303,3 +303,10 @@ If you are managing your cloud provider credentials with mint mode, the IAM user
 ====
 * `sts:AssumeRole`
 ====
+
+.Required permissions for enabling Bring your own public IP addresses (BYOIP) feature for installation
```

Comment:
I believe it is interesting to keep the IP version.

```suggestion
.Required permissions for enabling Bring your own public IPv4 addresses (BYOIP) feature for installation
```

Reply:
sure, done!